app/main/index.js

```diff
@@ -344,6 +346,11 @@ app.on('ready', () => {
 	ipcMain.on('realm-icon-changed', (event, serverURL, iconURL) => {
 		page.send('update-realm-icon', serverURL, iconURL);
 	});
+
+	ipcMain.on('error-reporting', event => {
+		const errorReporting = ConfigUtil.getConfigItem('errorReporting');
+		event.sender.send('error-reporting-val', errorReporting);
```

Comment:
Is there any reason for not using `page.send` here?

Reply:
Using page.send, the errorReporting value is given to the ```logger-util``` multiple times. When event.sender.send is used it is given only once, which should be the case as the logger is loaded only once on application start.

Reply:
Okay, maybe add this in the comment?

Reply:
Done.